lib/engine/game/g_1880/step/selection_auction.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../../step/passable_auction'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class SelectionAuction < Engine::Step::Base
```

Comment:
I believe selection auction already exists

Reply:
I looked at the code and I see that so far selection auction and draft only exist within games. Is this selection auction or the simple draft quite different from the others? If not, please add to `lib/engine/steps` and subclass for any minor differences. If they are quite different, then carry on.

Reply:
they are pretty similar. Will subclass